lib/modelValidator.js

```diff
@@ -387,6 +396,23 @@ function validateArray(name, value, field, models, allowBlankTargets, disallowEx
         }
         arrayType = fieldType;
     } else if(models && field.items && field.items.$ref) {
+        if (field.type && field.uniqueItems) {
+            var duplicatedObjects = []
+            for (var i = 0; i < value.length; i++) {
+                var j = i + 1
+                while(j < value.length && !doesArrayContainsValue(duplicatedObjects, value[j])) {
+                    if (_.isEqual(value[i], value[j])) {
```

Comment:
Any chance we can avoid bringing in lodash?  can we do an equality check ourselves?

Reply:
Perhaps we can use lodash.isEqual if thats all the lodash we need?

Reply:
two soultions :
- create function to compare object (both keys and values)
- just import lodash.isEqual which avoid to import the full lib

Reply:
I'd prefer we do the compare outselves but I'd accept the lodash.isEqual solution as well.

Reply:
Lodash.isEqual is probably the better solution - even though it means we have a dependency on the lodash.isEqual package.  I was hoping to have no dependencies if possible but comparing objects is kind of nasty so using a know good library is the best solution.  Please change lodash to lodash.isEqual and we can merge this.

Reply:
done

Reply:
Thanks